packages/grafana-ui/src/components/TimeSeries/utils.ts

```diff
@@ -327,7 +327,7 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<{ sync: DashboardCursor
       key: '__global_',
       filters: {
         pub: (type: string, src: uPlot, x: number, y: number, w: number, h: number, dataIdx: number) => {
-          payload.columnIndex = dataIdx;
+          payload.rowIndex = dataIdx;
```

Comment:
This seems to be an error with the existing tooltip, but I don't think it was getting used before